### PR TITLE
gift vouchers, v 0.7.0

### PR DESCRIPTION
**Pull request by LukaszWiktor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -584,6 +616,7 @@ Result:
 `This endpoint does not return result`
     
 ### Changelog
+- **2016-04-23** - `0.7.0` - Gift vouchers.
```
Comment (mandraszyk) [inline README.md]: why we are releasing 0.7.0 before 0.6.0...

File: VoucherBuilder.php [hunk 2]
```diff
@@ -50,6 +53,14 @@ public function setUnitDiscount($unit_off, $unit_type) {
                 return $this;
             }
             
+            public function setGiftAmount($amount) {
+                $this->_voucher->type = "GIFT_VOUCHER"
```
Comment (mandraszyk) [inline VoucherBuilder.php]: missing `;` at the end of line

File: README.md (hunk 1 above)
Comment (LukaszWiktor) [inline README.md]: image

File: VoucherBuilder.php (hunk 2 above)
Comment (LukaszWiktor) [inline VoucherBuilder.php]: thanks, fixed

**Pull request merged**